bears/matlab/MatlabIndentationBear.py

```diff
@@ -12,6 +12,7 @@ class MatlabIndentationBear(LocalBear):
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/97596'
```

Comment:
hey, you're running the bear on a python file, not on the matlab file. The python file with the tests contains code samples, for a proper asciinema you'll have to take the matlab code samples and put them in `.m` files, run the bear on those.

Let us know if you need any help with that!